manifests/g/GodotEngine/GodotEngine/4.1.1/GodotEngine.GodotEngine.locale.en-US.yaml

```diff
@@ -7,11 +7,39 @@ PackageLocale: en-US
 Publisher: Godot Engine
 PublisherUrl: https://godotengine.org
 PublisherSupportUrl: https://github.com/godotengine/godot/issues
+PrivacyUrl: https://godotengine.org/privacy-policy
+Author: Godot Engine
 PackageName: Godot Engine
-PackageUrl: https://github.com/godotengine/godot
+PackageUrl: https://godotengine.org/download/windows
 License: MIT License
-LicenseUrl: https://github.com/godotengine/godot/blob/master/LICENSE.txt
+LicenseUrl: https://godotengine.org/license
+Copyright: |-
+  Copyright (c) 2014-present Godot Engine contributors.
+  Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.
+CopyrightUrl: https://godotengine.org/license
 ShortDescription: Multi-platform 2D and 3D game engine
-ReleaseNotesUrl: https://github.com/godotengine/godot/releases/tag/4.1.1-stable
+Description: |-
+  Godot Engine is a feature-packed, cross-platform game engine to create 2D and 3D games from a unified interface.
+  It provides a comprehensive set of common tools, so that users can focus on making games without having to
+  reinvent the wheel. Games can be exported with one click to a number of platforms, including the major desktop
+  platforms (Linux, macOS, Windows), mobile platforms (Android, iOS), as well as Web-based platforms and consoles.
+Moniker: godot
+Tags:
+- game-development
+- game-engine
+- gamedev
+- godot
+- godotengine
+- hacktoberfest
+- multi-platform
+- open-source
+ReleaseNotes: Godot 4.1.1 is a maintenance release fixing a number of issues while preserving compatibility with the previous 4.1.x releases.
+ReleaseNotesUrl: https://godotengine.org/article/maintenance-release-godot-4-1-1
+InstallationNotes: |-

```

Comment:
Installation notes are only shown *after* install, so this won't help users

Reply:
> Installation notes are only shown _after_ install, so this won't help users

Is there a better way to get this info to users?

Reply:
> > Installation notes are only shown _after_ install, so this won't help users
> 
> Is there a better way to get this info to users?

Currently, no
* https://github.com/microsoft/winget-cli/issues/3483